backend/content-module/services/serviceContent.ts

```
@@ -665,12 +668,17 @@ export class ContentService {
 
       // If no creators found from liked content, get trending creators
       if (creatorUids.length === 0) {
-        console.log("No creators from liked content, getting trending creators");
+        console.log(
+          "No creators from liked content, getting trending creators"
+        );
         const trendingContent = await this.getTrendingContent(20);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)